IPython/core/debugger.py

```diff
@@ -303,7 +380,9 @@ def __init__(self, color_scheme=None, completekey=None,
 
         # list of predicates we use to skip frames
         self._predicates = self.default_predicates
+        self._skipping = False
```

Comment:
I think this was already covered in `self._predicates`?

Reply:
Yep you are right
```suggestion
```